docs/overview/migrating-react-native.md

```diff
@@ -5,35 +5,87 @@ sidebar_label: Migrating react-native
 original_id: migrating_react_native
 ---
 
-
-
 ReNative supports react-native projects pretty much out of the box in terms of your application code
 
 Due to variety of possibilities / complexity fo react-native projects (ie plugins, custom build scripts and so on) there might be some extra work required to reconnect existing plugins
 
 ---
+
 ## Migration Guide
 
 1. Install rnv
 2. Create new project `rnv new` (pick @rnv/template-starter)
-3. follow rest of the steps
-4. After first `rnv run ...` check out the codebase and pay attention to:
+3. Follow rest of the steps of new project installation.
+4. After first `rnv run ...` check out the codebase.
+
+### ReNative Project Folder Structure
+For more information about the project structure, please visit this <a href="/docs/overview/architecture#build-process">link</a>.
+
+The `src/` folder contains the source content for your application. The initial structure of the `src/` folder includes the following subfolders:
+
+-   `app`: This folder contains the main files for your application.
+-   `entry`: This folder allows you to configure the launch settings of your app. It is intended for advanced configuration and customization.
+-   `pages`: This folder enables you to create individual pages for your application.
```

Comment:
This should be stated, that is reserved for Next.js, if they don't do web, they don't need it, they can also specify a different folder in appConfigs/renative.json, but this is not just for individual pages for all platforms

Reply:
changed